oshdb-api/src/main/java/org/heigit/bigspatialdata/oshdb/api/mapreducer/MapReducer.java

```diff
@@ -1836,8 +1851,8 @@ protected TagTranslator getTagTranslator() {
 
   // hack, so that we can use a variable that is of both Geometry and implements Polygonal (i.e.
   // Polygon or MultiPolygon) as required in further processing steps
+  @SuppressWarnings("unchecked") // all setters only accept Polygonal geometries
```

Comment:
this is how I prefer it, comments above or in the some line. (see previous comment)

Reply:
I also prefer this version, but sometimes the line becomes too long for the space left after the annotation, then I just chose to put it in a new line, so the comment is without a line break (thus, slightly easier to read): I prefer `@Suppress… \n // long bla` over `@Suppress… // long \n // bla`. But again, this might just be a matter of personal taste.